docs/api/system.md

```diff
@@ -0,0 +1,90 @@
+# System
+
+System APIs allows you to interact with dialogs in the system-level (e.g. permissions, alerts, etc.).
+
+:::caution Experimental
+
+System APIs are currently in an experimental phase.
+This means that the API is not yet stable and may change in the near future.
+**Do not rely on the current API as it may change in future minor releases.**
```

Comment:
This line `Do not rely on this API` sounds almost like shooting in your own leg. Needs rephrasing. Maybe just mention that "This API may change over minor releases", because "Do not rely" sounds like "Please don't use it :) "